exotica/src/Scene.cpp

```diff
@@ -550,7 +559,7 @@ void Scene::addTrajectory(const std::string& link, std::shared_ptr<Trajectory> t
     const auto& it = tree.find(link);
     if (it == tree.end()) throw_pretty("Can't find link '" << link << "'!");
     if (traj->getDuration() == 0.0) throw_pretty("The trajectory is empty!");
-    trajectory_generators_[link] = std::pair<std::shared_ptr<KinematicElement>, std::shared_ptr<Trajectory>>(it->second.lock(), traj);
+    trajectory_generators_[link] = std::pair<std::weak_ptr<KinematicElement>, std::shared_ptr<Trajectory>>(it->second, traj);
```

Comment:
This need to be fixed/updated in updateSceneFrames or else loading another scene can break trajectory-generated objects